zui/src/setup.test.ts

```diff
@@ -18,6 +19,14 @@ export function isValidTypescript(
   }
 }
 
+export function evalZod(script: string): z.ZodType {
```

Comment:
**selfnote:** an equivalent function is already declared in `src/transforms/json-schema-to-zui/index.ts`. Extract in common location